pykernels/graph/randomwalk.py

```diff
@@ -8,7 +8,7 @@
 from pykernels.base import Kernel, GraphKernel
```

Comment:
Fair enough, I don't have strong feelings either way, it is just the previous import scheme was broken in Python 3.